addons/medical_treatment/functions/fnc_overDose.sqf

```diff
@@ -0,0 +1,43 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Cplhardcore
+ * Handles the overdose effects of a medication.
+ *
+ * Arguments:
+ * 0: The patient <OBJECT>
+ * 1: Medication classname <STRING>
+ * 2: Incompatible medication <ARRAY of STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "morphine"] call ace_medical_treatment_fnc_overDose
+ * Public: No
+ */
+
+params ["_unit", "_classname"];
+
+private _medicationConfig = configFile >> "ace_medical_treatment" >> "Medication";
+private _onOverDose = getText (_medicationConfig >> "onOverDose");
+
+if (isClass _medicationConfig) then {
+    _medicationConfig = _medicationConfig >> _classname;
+    if (isText (_medicationConfig >> "onOverDose")) then { 
+        _onOverDose = getText (_medicationConfig >> "onOverDose"); 
+    };
+};
+TRACE_2("overdose",_classname,_onOverDose);
+
+if (_onOverDose == "") exitWith {
+    TRACE_1("CriticalVitals Event",_unit);
+    [QEGVAR(medical,CriticalVitals), _unit] call CBA_fnc_localEvent;
+};
+
+if (!isNil _onOverDose) then {
```

Comment:
This check in general doesn't make sense - in the updated form you are trying to get a mission variable called `nil`.

Reply:
it has been reverted to as it was before

Reply:
Sorry, I thought it was code you added. It makes no sense and needs to be fixed, but we'll do that in another PR.